content/Store Support/Shipping/About-the-delivery-process.md

```diff
@@ -87,19 +87,30 @@ If you want to return your order and receive a refund you can reject to receive
 
 ---
 
-<a id="if-a-parcel-did-not-arrive"></a>
+<a id="if-you-havent-received-your-delivery"></a>
 
-## If a parcel did not arrive
+## If you haven't received your delivery
 
-In some rare cases, an order might not be delivered to the customer even if it appears as delivered when tracking the parcel. This mostly happens when using shipping methods that don't have proof of delivery.
+In rare cases, an order might not reach its destination even when the tracking displays it as delivered. This is more common with shipping methods that don't offer proof of delivery.
 
-If the parcel did not arrive, please check the following before contacting [Arduino's customer support](https://www.arduino.cc/en/contact-us/):
+Here are a few steps to follow if a delivery is missing:
 
-* Check if other people living in the same delivery address or a neighbor have picked up the parcel.
-* Check around the entrance(s) of your residence, particularly on back porches, bushes, garages, grills, or other places that might protect your package from theft or weather.
+1. See if someone else at your delivery address or a nearby neighbor have picked up the parcel.
+
+1. Check for messages or updates from the shipping provider to see if your package might have been dropped off at a pickup point. You can also review the tracking information provided by the shipping partner.
+
+    ![Image of a link that leads to shipping informaiton](img/delivery_link.png)
+
+1. Ensure your address details are accurate and up to date.
+
+1. Check around the entrance(s) of your residence, particularly on back porches, bushes, garages, grills, or other places that might protect your package from theft or weather.
 
 > Note that carriers may also use plastic bags to protect your package from the weather.
 
+### What to do next?
```

Comment:
Please create this as a new article :)

I created a draft in Zendesk Guide to generate an article id that you can use: 14085643116700

You can replace the text in this section with a link (https://support.arduino.cc/hc/en-us/articles/14085643116700) using the same pattern as in "Track your order"